terraform/.terraform.lock.hcl

```diff
@@ -42,21 +42,22 @@ provider "registry.terraform.io/hashicorp/archive" {
 }
 
 provider "registry.terraform.io/hashicorp/aws" {
-  version     = "4.0.0"
+  version     = "3.74.1"
```

Comment:
The version has dropped here should it go up to 4.1.0 as seen in the management-account/terraform/.terraform.lock.hcl file ?

Reply:
PR #355 updated it to 4.0.0, which it shouldn't have (should have been 3.74.2) so undoing that 👍🏼